lib/ddtrace/contrib/http/patcher.rb

```diff
@@ -12,6 +12,12 @@ module HTTP
       APP = 'net/http'.freeze
       SERVICE = 'net/http'.freeze
 
+      @distributed_tracing_enabled = false
```

Comment:
@palazzem I added this one because manipulating the Pin at each client call was, IMHO, really a pain at some point. For the good and the bad, this enables/disables it globally. It's harder to find a proper grain for that config with Net/HTTP (client lib) than with the server side (Rack, for instance) as there's no such thing as a "global client instance". While there's a global server instance and config.